AirCasting/ABConnector/BluetoothManager.swift

```diff
@@ -159,6 +163,10 @@ extension BluetoothManager: CBPeripheralDelegate {
         }
     }
     
+    func disconnectAirBeam() throws {
+        mobilePeripheralSessionManager.finishActiveSession(centralManger: centralManager)
+    }
+    
```

Comment:
Why this is a throwing function?

Reply:
You're right, it doesn't have to be throwing, changing that :)